gui/widgets/popup.cpp

```diff
@@ -387,14 +391,35 @@ void PopUpDialog::drawMenuEntry(int entry, bool hilite) {
 
 	Common::String &name(_entries[entry]);
 
+	Common::Rect r1(x, y, x + w, y + _lineHeight);
+	Common::Rect r2(x + 1, y + 2, x + w, y + 2 + _lineHeight);
+	Graphics::TextAlign alignment = Graphics::kTextAlignLeft;
+
+	if (g_gui.useRTL()) {
+		if (_twoColumns) {
+			r1.translate(this->getWidth() - w, 0);
+		}
+
+		r2.left = g_system->getOverlayWidth() - r2.left - w + g_gui.getOverlayOffset();
+		r2.right = r2.left + w;
+
+		if (_boss->getFlags() == 305) {
```

Comment:
What is `305`? Please use the constants, not a magic value.

Reply:
I haven't figured out a way to center align texts from dropdownbutton widgets. I've left a FIXME note just below this line for now

Reply:
this was wrong code, now removed